website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -67,9 +67,9 @@ To set your job's schedule, use the **Run on schedule** option to choose specifi
 
 Under **Timing**, you can either use customizable hours for jobs that need to run frequently throughout the day or exact intervals for jobs that need to run at specific times:
 
-- **Exact intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
+- **Intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
```

Comment:
```suggestion
- **Intervals** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC (plus account-level jitter). This option is useful if you need to run jobs multiple times per day at regular intervals.
```

Reply:
I would try and do this as an example rather than saying jitter because I'm not sure jitter is an external thing people will know. What about something like:
For example, if you set it to **Every 2 hours**, the job will run every 2 hours from midnight UTC. Note this does not mean that it will run at midnight UTC exactly. However, subsequent runs will always be run at the same time. For example, if the previous scheduled pipeline ran at 00:04 UTC, the next run will be at 02:04 UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.

Reply:
thanks for the copy! reworded it a bit for clarity. lemme know what you think